mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/repositories/TiltaksgjennomforingRepository.kt

```diff
@@ -173,6 +173,16 @@ class TiltaksgjennomforingRepository(private val db: Database) {
         val order = when (filter.sortering) {
             "navn-ascending" -> "tg.navn asc"
             "navn-descending" -> "tg.navn desc"
+            "tiltaksnummer-ascending" -> "tg.tiltaksnummer asc"
+            "tiltaksnummer-descending" -> "tg.tiltaksnummer desc"
+            "arrangor-ascending" -> "tg.virksomhetsnummer asc"
+            "arrangor-descending" -> "tg.virksomhetsnummer desc"
```

Comment:
Denne blir også merkelig å sortere på for øyeblikket

Reply:
e9